src/handlers/deeplinks.ts

```diff
@@ -91,6 +91,12 @@ export default async function handleDeeplink(
        * Universal links from WC e.g. when initiating a pairing on mobile, you
        * tap "Rainbow" in Web3Modal and it hits this handler
        */
+      case 'ethereum': {
+         logger.info(`handleDeeplink: ethereum`);
+         const normalizedUrl = url.replace('rainbow://ethereum', 'ethereum:/');
```

Comment:
@brunobar79 should this be `url.replace('rainbow://ethereum', 'ethereum://');` (double forward slash in last argument?)

Reply:
already fixed